apps/recovery-portal/src/main/webapp/password-recovery.jsp

```diff
@@ -288,11 +294,11 @@
 
                 var reCaptchaResponse = $("[name='g-recaptcha-response']")[0].value;
 
-                if (reCaptchaResponse.trim() == '') {
+                if (reCaptchaResponse.trim() === '') {
```

Comment:
```suggestion
                if (reCaptchaResponse.trim() === "") {
```

Reply:
Fixed the equality check, in commit f85c0f0